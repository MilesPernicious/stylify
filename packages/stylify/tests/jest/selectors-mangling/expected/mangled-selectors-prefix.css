.se:hover{
	color: red
}
.sf{
	color: blue
}
.sb:focus{
	color: orange
}
.sa a{
	color: green
}
.sc{
	color: yellow
}
.sd{
	font-size: 24px
}
