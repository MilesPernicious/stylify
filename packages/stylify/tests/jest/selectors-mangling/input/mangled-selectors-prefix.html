<!--
stylify-components
	button: `
		color:yellow
		&--big {
			font-size:24px
		}
	`
/stylify-components
-->
<div class="
	color:blue
	hover:color:red
	focus:{color:orange}
	[a]{color:green}
"></div>

<div class="button button--big"></div>
