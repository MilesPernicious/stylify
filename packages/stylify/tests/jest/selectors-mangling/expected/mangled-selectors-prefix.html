<!--
stylify-components
	button: `
		color:yellow
		&--big {
			font-size:24px
		}
	`
/stylify-components
-->
<div class="
	sf
	se
	sb
	sa
"></div>

<div class="sc sd"></div>
